Update links to DigiNode Tools website - https://diginode.tools

saltedlolly opened the pull request:
Remove links to Github repo.

ycagel commented:
@saltedlolly Why is this PR in the DigiByte.org website repo?

JaredTate reviewed (approved):
cACK. Thanks for all the effort you have put into this platform @saltedlolly

ycagel reviewed (approved):
cACK

ycagel commented:
@saltedlolly It looks like you changed one of Renzo's links. This doesn't seem right?

![image](https://github.com/DigiByte-Core/DGBio/assets/33166838/3b6e3ed3-1646-45b8-bcf6-6fe51c2caf8f)

saltedlolly commented:
> @saltedlolly It looks like you changed one of Renzo's links. This doesn't seem right? Thanks for finding this @Jongjan88.
> 
> ![image](https://private-user-images.githubusercontent.com/33166838/318062673-3b6e3ed3-1646-45b8-bcf6-6fe51c2caf8f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3MjE3MTIsIm5iZiI6MTcxMTcyMTQxMiwicGF0aCI6Ii8zMzE2NjgzOC8zMTgwNjI2NzMtM2I2ZTNlZDMtMTY0Ni00NWI4LWJjZjYtNmZlNTFjMmNhZjhmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI5VDE0MTAxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFmZDZhMjVhZDZkMzZkNGQ4ZDkyOWQ1NTNhMGVjZjQ2NmM1NmM2MTI5YTFmYmJlNmYxMmFiYWM5ODYwZTk2ODEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KrCRiLlkUGZcLSzGTm5n_iNhPX5ul4vMzKp4kgG6r88)

Nice catch. I had changed the wrong one by mistake. I have fixed it.

Sbosvk reviewed (approved):
utACK
